common/libs/README.md

```diff
@@ -0,0 +1,2 @@
+The `numpy.js` file under this folder is copyed from https://github.com/lutzroeder/netron/blob/main/source/numpy.js
```

Comment:
a typo: copyed -> copied

Please also specify the commit id of netron where you copied numpy.js from.